css/interface.css

```diff
@@ -11,7 +11,7 @@ body {
   right: 15px;
   left: 15px;
   margin: 0;
-  z-index: 1;
```

Comment:
why 3? worth making a comment?

Reply:
The interface contians the `.select-proxy-display` dropdown, for which the `.select-proxy-display .icon` icon element has a `z-index` of 2

Reply:
This means a `z-index` of 3 is required to ensure the alert message sits on top of everything else